app/models/recognized-worship-type.js

```diff
@@ -0,0 +1,6 @@
+import Model, { attr } from '@ember-data/model';
+
+export default class RecognizedWorshipType extends Model {
```

Comment:
If the relationship is removed, this can probably go as well?

Reply:
I think yes since only the route is called, I'll quickly check

Reply:
Works without it